W-17800939 Feedback round 1 (#775)
mulesoft/docs-gateway: gateway/1.9/modules/ROOT/pages/flex-gateway-version-lifecycle.adoc

|===

For detailed information and support timeframes for Flex Gateway 1.8 and earlier, see the https://www.salesforce.com/content/dam/web/en_us/www/documents/legal/Agreements/versioning-back-support-policy.pdf[MuleSoft Support Policy]. 

== Deployment Models and Lifecycle Management

Flex Gateway lifecycle management varies depending on the release channel and the type of gateway. Managed gateways are auto-upgraded regularly, and any available patches are applied once a month. Self-managed gateways are never patched or upgraded automatically.

[%header,cols="3*a"]
|===
| Action | Managed Gateway | Self-Managed Gateway

// Action
| Auto-upgrade
// Managed Gateway
a| 
Managed Gateways are auto-upgraded to the latest minor version within their respective channels (Edge or LTS).

    * Edge: Three times per year
    * LTS: Once a year
// Self-Managed Gateway
| No auto-upgrade capabilities

// Action
| Patches
// Managed Gateway
| Monthly auto-patching
// Self-Managed Gateway
| Available for Flex Gateway versions that are within the support period for their respective channels (Edge or LTS).

// Action
| Self-upgrade
// Managed Gateway

> | You have the option to manually upgrade to the latest version that matches your release channel (Edge or LTS) before the scheduled automatic upgrade.

Reviewer: ```suggestion
| You can manually upgrade to the latest version that matches your release channel (Edge or LTS) before the scheduled automatic upgrade.
```
